command/ca/revoke.go

```diff
@@ -405,11 +407,22 @@ func (f *revokeFlow) Revoke(ctx *cli.Context, serial, token string) error {
 	if len(token) == 0 {
 		certFile, keyFile := ctx.String("cert"), ctx.String("key")
 
-		// If there is no token then we must be doing a Revoke over mTLS.
-		var cert tls.Certificate
-		cert, err = tls.LoadX509KeyPair(certFile, keyFile)
+		certPEMBytes, err := ioutil.ReadFile(certFile)
 		if err != nil {
-			return errors.Wrap(err, "error loading certificates")
+			return errors.Wrap(err, "error reading cert")
```

Comment:
As this will become a user error is better to write `certificate` instead of `cert`